### PR TITLE
Add a Help section

### PR DESCRIPTION
**Pull request by Regisle**
Just a simple help file containing hotkeys shortcuts and other lesser known features

Any and all info to add is helpful as theres alot of obscure stuff

### CODE REVIEW

**Comment by Regisle** [comment]
other hotkeys:
 shift, pause = start/stop profiler
 normal copy/paste hotkeys work, also works for items and skills

**Comment by QuickStick123** [comment]
In Simple Graphic:
Console – Has full normal console controls e.g. autocomplete on tab, history arrows, etc
Printscreen - Should screenshot but broken error msg "Couldn't write screenshot!"
Pause - Toggles enable of profiling.
Crt + ` toggles console
In POB:
In Dev Mode:
During reload/refresh
Ctrl - Rebuild Mod cache
Crtl + Shift - Allow Tree Download
Alt - Show Advance mod breakdown/Passing.
F5 - Restart
F6 - Force Run Garbage collector
Shift - Exporting build while holding shift will save xml to clipboard.
No Dev mode:
Ctrl + u - Check for update.
Ctrl + z - undo
Ctrl + y - redo
WHEELUP/DW or arrows - scrolls
Ctrl + c - copy
Ctrl + x - cut
Ctrl + v  or right mouse (over text box) - paste
Ctrl + a - select all
Regular text hotkeys, home, end, delete, arrows all supported.
Backspace/Delete + Ctrl - Faster character removal.
Escape - close / return to previous menu
return/enter - confirm thing.
Tab - Doesn’t Work?
Ctrl + +/- and 0 - behave normal for zoom.
When creating an items either through item creator or adding an item pressing ctrl will add it to the first slot and ctrl shift will add it to the second slot e.g. offhand for a weapon.
F2 - Rename items set, etc.
F1 - Open item/gem/etc in poewiki.net
e - on an equipped item will open it on the edit menu on the right.
ctrl + f - brings up search boxes e.g. unique item / tree.
Ctrl - Hides tooltip on tree.
p - toggle node power
Ctrl + d - toggles stat differences (e.g. dps gained on node select)
pgUP/DW or Scroll - zoom on tree shift augments x3.
Ctrl + LeftMB - zoom on cursor.
Holding Shift - enable “path trace mode” on tree that keeps the current planned path highlighted but lets you move the cursor away and changes some of the node counts to nodes in path wording.
Ctrl + LeftMB - Enable/Disable Gems
Ctrl + RightMB - Enable/Disable Gems from Full DPS 
Mouse4 or Ctrl + w- Close Build (gives save prompt if unsaved.)
Ctrl + s - save file
Ctrl + i - jump to import tab
Ctrl + 1 - jump tree tab
Ctrl + 2 - jump skills tab
Ctrl + 3 - jump items tab
Ctrl + 4 - jump calcs tab
Ctrl + 5 - jump config tab
Ctrl + 6 - jump notes tab
Ctrl + n - new build ( build selection menu )
Mouse4/5 - undo redo path respectively (build selection menu)

**Comment by Nightblade** [comment]
more hotkeys:

* `ctrl+z` to undo, `ctrl+y` to redo
* `ctrl +/-` or `Ctrl + mouse wheel` to "zoom" (change text size) in Notes tab.

| Key Combo | Action |
|---------------|---------|
| `ctrl+6`        | switch to Notes tab |

@QuickStick123 is indeed faster than me!

File: help.txt [hunk 1]
```diff
@@ -6,13 +6,21 @@ This file contains a list of shortcuts, and other misc things PoB has
...
+4. Hotkeys
+5. Other
+6. Timeless Jewels
 
----[Hotkeys][3]
+---[FAQ][3]
 
+---[Hotkeys][4]
+
+ctrl, ` = dev consol
```
Comment (Nightblade) [inline help.txt:18]: `consol` -> `console`

File: help.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+When creating an item either through item creator or adding an item pressing ctrl will add it to the first slot and ctrl+shift will add it to the second slot e.g. offhand for a weapon.
+
+Passive Tree Shortcuts:
+
+ctrl	Hide tooltips while held
+p	Toggle node power
+ctrl+d	Toggle stat diff display
+PGUP/PGDN/MWHEEL	Zoom in/out (hold with shift to increase amount of zoom x 3)
+ctrl+LMB	Zoom in on mouse cursor position
+hold shift	Enable "path trace mode" (show shortest path to mouse-overed-node)
```
Comment (QuickStick123) [inline help.txt:58]:
```suggestion
hold shift	Enable "path trace mode" (nodes highlighted will be added to a path which will be allocated when the final node is clicked on the tree)
```

**Review by LocalIdentity — APPROVED** [review]

**Pull request merged**